core/unix/os.c

```diff
@@ -5035,7 +5035,12 @@ ignorable_system_call_normalized(int num)
     case SYS_readlinkat: return !DYNAMO_OPTION(early_inject);
 #endif
 #ifdef SYS_openat2
-    case SYS_openat2: return IS_STRING_OPTION_EMPTY(xarch_root);
+    case SYS_openat2:
+        if (!IS_STRING_OPTION_EMPTY(xarch_root))
+            return false;
+        SYSLOG_INTERNAL_WARNING_ONCE(
+            "WARNING i#5131: possibly unhandled system call openat2");
```

Comment:
I think this should be removed: we don't care about open and so are unlikely to care about openat2.

Reply:
But we do care about `SYS_openat`, which is why I assumed we should atleast warn for `SYS_openat2`. Xref the discussion in the comment thread: IIUC we expect any issues with openat2 handling to be clearly visible, and that is why we want to remove the warning?

Reply:
If -xarch_root is not set I don't think there should be any warning.

Reply:
Removed it.